### PR TITLE
feat: add change date time format dialog

### PR DESCRIPTION
**Pull request by FunkyMuse**

### CODE REVIEW

File: commons/src/main/kotlin/com/simplemobiletools/commons/dialogs/ChangeDateTimeFormatDialog.kt [hunk 1]
```diff
@@ -71,3 +95,196 @@ class ChangeDateTimeFormatDialog(val activity: Activity, val callback: () -> Uni
...
+        ) {
+            var is24HoursSelected by remember { mutableStateOf(is24HourChecked) }
+
+            val (selected, setSelected) = remember { mutableStateOf(initiallySelected) }
+            RadioGroup(
+                items = kinds, selected = selected,
+                setSelected = setSelected,
+                modifier = Modifier.padding(
+                    vertical = 16.dp,
+                    horizontal = 11.dp
```
Comment (tibbi) [inline commons/src/main/kotlin/com/simplemobiletools/commons/dialogs/ChangeDateTimeFormatDialog.kt:146]: lets use radiobuttons that are reusable in other dialogs and apps too, do not hardcode values here

**Comment by tibbi** [comment]
do not use caps lock at February etc

**Comment by tibbi** [comment]
that tiny ripple effect around text itself looks weird, maybe you can add some padding around it?

**Comment by tibbi** [comment]
lets also change the sample year from 2021 to 2023

**Comment by FunkyMuse** [comment]
> lets also change the sample year from 2021 to 2023

February 02, 2023 makes sense?

**Comment by tibbi** [comment]
no, make it february 15 like now, the day is above 12 intentionally to avoid confusion

**Comment by tibbi** [comment]
looks better now, but reduce the margin between radiobuttons to like 2dp line only. And make the ripple stretco to full width to make the top and bottom checkbox consistent

**Comment by FunkyMuse** [comment]
> looks better now, but reduce the margin between radiobuttons to like 2dp line only. And make the ripple stretco to full width to make the top and bottom checkbox consistent

checkbox isn't full width ripple

**Comment by tibbi** [comment]
thats what I mean though, lets make it full width

**Comment by FunkyMuse** [comment]
> thats what I mean though, lets make it full width

Can you check now please

**Comment by tibbi** [comment]
yeah, it is a lot better now, try horizontally aligning the radio circles and bottom checkbox. Also in landscape mode the whole dialog scrolls including Cancel/OK. Shouldnt those buttons be always visible and only the content should scroll? Thats how it worked previously and I prefer it, always seeing Cancel/OK.

**Comment by FunkyMuse** [comment]
> yeah, it is a lot better now, try horizontally aligning the radio circles and bottom checkbox. Also in landscape mode the whole dialog scrolls including Cancel/OK. Shouldnt those buttons be always visible and only the content should scroll? Thats how it worked previously and I prefer it, always seeing Cancel/OK.

Done, also added star rating dialog

**Comment by tibbi** [comment]
yeah, looks great. One more details, can we show the scrollbar for a moment on the side, if content actually scrolls? It happened in the past that in the gallery sorting dialog people havent noticed that there is an additional checkbox at the bottom, because they had no idea that it actually scrolls. Maybe this small indication would help a bit.

**Comment by tibbi** [comment]
as per discussion, compose doesnt support it properly yet, so accepting it

**Pull request merged**